src/test/java/edu/uci/ics/cs221/index/Team2StressTest/Team2StressTest.java

```diff
@@ -0,0 +1,149 @@
+package edu.uci.ics.cs221.index.Team2StressTest;
```

Comment:
All test cases should be in the package `edu.uci.ics.cs221.index.inverted`.
See [TA's instruction on Slack](https://uci-cs221-s19.slack.com/archives/CH7JR156D/p1556138044128600).
```suggestion
package edu.uci.ics.cs221.index.inverted;
```
and move this file to `.../index/inverted/` accordingly.

Reply:
Thanks. I have updated the file location.